### PR TITLE
fix: Delete multi-region review comment in values.yaml

### PR DESCRIPTION
**Pull request by hamza-m-masood**
### Which problem does the PR fix?

Related to: https://github.com/camunda/distribution/issues/242
### What's in this PR?

I have deleted the comment in the values.yaml for the review of multiregion setup for customers.
### Checklist

Please make sure to follow our [Contributing Guide](../blob/main/CONTRIBUTING.md).

**Before opening the PR:**

- [x] In the repo's root dir, run `make go.update-golden-only`.
- [x] There is no other open [pull request](../pulls) for the same update/change.
- [x] Tests for charts are added (if needed).
- [x] In-repo [documentation](../blob/main/CONTRIBUTING.md#documentation) are updated (if needed).

**After opening the PR:**

- [x] Did you sign our CLA (Contributor License Agreement)? It will show once you open the PR.
- [ ] Did all checks/tests pass in the PR?

### CODE REVIEW

**Comment by hamza-m-masood** [comment]
@hisImminence Should I also remove this line?:
```
Camunda reserves the right to limit support if no review was done prior to launch or the review showed significant risks.
```

Will Camunda still be liable to support the customer if the setup is not fully correct?

**Comment by hamza-m-masood** [comment]
@hisImminence I reflected the changes in the docs and kept this last line since I think it makes sense to keep it: 
```
# If you can, consider three regions.
```

**Review by hisImminence — APPROVED** [review]
Thank you @hamza-m-masood!

File: charts/camunda-platform/values.yaml [hunk 1]
```diff
@@ -322,11 +322,10 @@ global:
 
   # Multiregion options for Zeebe
   #
-  ## WARNING: In order to get your multi-region setup covered by Camunda enterprise support you MUST get your configuration and run books reviewed by Camunda before going to production.
-  # This is necessary for us to be able to help you in case of outages, due to the complexity of operating multi-region setups and the dependencies to the underlying Kubernetes prerequisites.
+  ## WARNING: Before advancing to production go-live, validating these procedures with Camunda is strongly recommended.
```
[the remark this answers is not in the document]
Comment (hisImminence) [reply charts/camunda-platform/values.yaml]: Change was made

**Pull request merged**